java/registry/src/main/java/dev/sunbirdrc/registry/helper/RegistryHelper.java

```diff
@@ -350,17 +350,17 @@ public JsonNode readEntity(JsonNode inputJson, String userId) throws Exception {
      * @return
      * @throws Exception
      */
-    public JsonNode searchEntity(JsonNode inputJson, String userId) throws Exception {
-        return searchEntity(inputJson, searchService, userId);
+    public JsonNode searchEntity(JsonNode inputJson, String userId, boolean forceNativeSearch) throws Exception {
```

Comment:
why did we add this flag ?

Reply:
To reuse this function to be able to search with native search as well when elastic search is enabled. with this reuse the function `searchEntitiesByUserId` is removed.